specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4242,16 +4246,20 @@
 		"properties": {
 			"name": {
 				"type": "string",
-				"description": "The name of the output."
+				"description": "The name path of the output."
```

Comment:
The name of the output defined by the skill.

Reply:
Fixed